src/scripts/main.js

```diff
@@ -1,3 +1,26 @@
 'use strict';
 
-// write code here
+const headElement = document.querySelector('thead');
+const tableElement = document.querySelector('tbody');
+
+const clickOnHeaders = () => {
+  headElement.addEventListener('click', () => {
```

Comment:
The event listener is attached to the entire `thead` element. If the table header contains multiple columns, it might be better to attach the event listener to specific header cells to avoid unexpected behavior.